### DISCUSSION TITLE
Give access on one repo to all members of an organization

### DISCUSSION THREAD

**Discussion by fastcatch** [community/community#23250] [Enterprise]
<p>Is there a way to designate a single repo to be writeable for all members of the organization (w/o setting a base permission that affects all repos)?  It looks to be a fairly basic requirement and thus it must be there somewhere just that I cannot find it. :frowning:</p>
<p>Having/Setting up an automatic “Everyone” team (see the <a href="https://github.community/t/how-to-create-a-team-containing-all-members-of-an-org/1713">How to create a team containing all members of an org?</a> issue) would be a way, but it seems to be “pending”. Adding people individually won’t cut it, because of the maintenance burden.</p>
<p>What is the doable way?</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/fastcatch">@fastcatch</a>,</p>
<p>Thank you for being here! Team synchronization is available for organizations using GitHub Enterprise Cloud.</p>
<p>Check out: <a href="https://developer.github.com/v3/teams/members/#add-or-update-team-membership" rel="nofollow noopener">https://developer.github.com/v3/teams/members/#add-or-update-team-membership</a> for more info.</p>
<p>Otherwise, as you mentioned, you could add all members of your org to a team using our API see: <a href="https://developer.github.com/v3/teams/#create-team" rel="nofollow noopener">https://developer.github.com/v3/teams/#create-team</a> and then </p>
<p>via the one single repository’s settings, or using our API add that team’s permissions to the read only or single repo: <a href="https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/managing-team-access-to-an-organization-repository" rel="nofollow noopener">https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/managing-team-access-to-an-organization-repository</a>. That would allow you to simply add another user to the team in future to give them the same level of access.</p>
<p>I hope this helps!</p>

**Reply by exonomyapp** [reply]
Hello Andrea, how do I give access to my personal repo to one team defined by a new organization I just created?

**Comment by suvl** [comment]
how do I add all current and future members of the org? regardless of team management, for one single repo, all members could write.
